### DISCUSSION TITLE
TikTokMulti v1.2.5

### DISCUSSION THREAD

**Discussion by Johnserf-Seed** [Johnserf-Seed/TikTokDownload#82] [General]
## TikTokMulti v1.2.5
**🎈欢迎提交你遇到的bug**

1. 新增长短链接识别功能 #79 
2. 新增命令行批量下载功能 #79 

3. 新增TKM GUI版本（还处在开发阶段）
4. 统一了控制台输出日志的风格
5. 修改了一些细节问题
6. 重写了PyQt鼠标的移动事件

![image](https://tvax2.sinaimg.cn/large/006908GAly1gyuycwma5lj30ux0qstiz.jpg)

**Full Changelog**: https://github.com/Johnserf-Seed/TikTokDownload/compare/1.2.3...v1.2.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnserf-Seed/TikTokDownload/releases/tag/v1.2.5'>TikTokMulti v1.2.5</a>.</em>

**Comment by Kaiwangkkk** [comment]
有没有可能设置下 下载一定时间内的用户短视频，比如2020.01.02-2021.01.02这种

**Reply by Johnserf-Seed** [reply]
这个建议可以采纳🎉
